pkgs/tools/misc/mat2/default.nix

```diff
@@ -0,0 +1,48 @@
+{ lib
+, poppler_gi
+, gdk-pixbuf
+, librsvg
+, ffmpeg
+, exiftool
+, python3
+, gobject-introspection
+, wrapGAppsHook
+}:
+
+let
+  inherit (python3.pkgs) mutagen pygobject3 pycairo buildPythonApplication fetchPypi;
+in buildPythonApplication rec {
+  pname = "mat2";
+  version = "0.11.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "beb99965eb41ebe022b6abed941aa0d23882e26b4b4b48f94295911a637f90ff";
+  };
+
+  # avoids AttributeError: module 'libmat2.archive' has no attribute 'ZipParser'
+  doCheck = false;
+
+  nativeBuildInputs = [ wrapGAppsHook ];
+  propagatedBuildInputs = [
+    mutagen
+    pycairo
+    pygobject3
+    librsvg
+    ffmpeg
+    exiftool
+    gobject-introspection
+    poppler_gi
+    gdk-pixbuf
+  ];
+
+  meta = with lib; {
+    description = "metadata removal tool supporting a wide range of commonly used file formats";
+    homepage = "https://0xacab.org/jvoisin/mat2";
+    license = licenses.gpl3;
```

Comment:
The gpl license variants without the `Plus` or `Only` suffix are deprecated, because they do not specify the license well enough. Use `Plus` if the project has "or later" sections in the actual source code files (not the license). Otherwise, use the `Only` suffix.
 
See https://discourse.nixos.org/t/lib-licenses-gpl3-co-are-now-deprecated/8206 for more details.